frontends/smvparser.y

```diff
@@ -250,7 +279,7 @@ constant: boolean_constant {
           | integer_constant {
```

Comment:
Also, what about boolean symbols? I.e. a `VAR` that's type `boolean`?

Reply:
Hi Makai,

I think I supported boolean value TRUE and FALSE using boolean constant in line 348. I can add another type SMVnode named boolean. Does it work better?

Reply:
Oh I see — I missed it because it was collapsed since it wasn’t involved in the diff. I think it’s fine how it is. Except, how do you handle VARs of type Boolean without a Boolean SMVNode?